cli/js/40_testing.js

```diff
@@ -503,17 +503,6 @@ function testStepPostValidation(desc) {
       );
     }
   }
-
-  // check if an ancestor already completed
-  let currentDesc = desc.parent;
-  while (currentDesc != null) {
-    if (MapPrototypeGet(testStates, currentDesc.id).finalized) {
-      throw new Error(
-        "Parent scope completed before test step finished execution. Ensure all steps are awaited (ex. `await t.step(...)`).",
-      );
-    }
-    currentDesc = currentDesc.parent;
-  }
```

Comment:
Why remove this?

Reply:
https://github.com/denoland/deno/pull/18246#discussion_r1140919178